docs/content/guide/quick-start.md

```
@@ -17,6 +17,88 @@ cd vetta
+
+```bash  
+atlas local setup vetta-local \  
+  --type local \  
+  --port 27017 \  
+  --bindIpAll \  
+  --skipMongosh  
+```  
+
+On first run the CLI pulls the required container images. Once ready you'll see:
+
+```  
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)